src/engraving/rw/compat/compatutils.cpp

```diff
@@ -560,3 +564,43 @@ void CompatUtils::resetStemLengthsForTwoNoteTrems(MasterScore* masterScore)
         }
     }
 }
+
+void CompatUtils::replaceStaffTextWithCapo(MasterScore* score)
+{
+    TRACEFUNC;
+
+    std::vector<StaffTextBase*> oldCapoList;
+
+    for (Measure* measure = score->firstMeasure(); measure; measure = measure->nextMeasure()) {
```

Comment:
It looks like this method does not handle linked parts. Either you need to iterate through the linked elements, like in `replaceStaffTextWithPlayTechniqueAnnotation` (and then you need to use a std::set because you might encounter the same element twice), or you need to use `score->scoresList()` to loop through all part scores.

Reply:
Addition: I think iterating through the elements might be the best choice. 

I'm namely afraid that there is no guarantee that `text->capo()` is synced between excerpts, so if we do the check `text->capo() > 1` for each part individually, the item might be replaced in some excerpts and not in others, which is a potentially dangerous situation. 

If you make the decision only once for the whole group of linked elements, then you are at least sure that either all elements of the group will be replaced, or none, so the situation stays consistent.

Reply:
thanks! fixed